src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+div {
+  margin: 20px 0;
+}
+
+.firstSearchString {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  position: relative;
+  width: 100%;
+}
+
+.firstInputLine {
+  width: 100%;
+  border: none;
+  outline: none;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-family: "Avenir-Heavy", Arial, Helvetica, sans-serif;
+  font-weight: 300px;
+  padding: 25px 0 23px 62px;
+}
+
+.firstInputLine::placeholder {
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: 300px;
+
+}
+
+.firstSearchString:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-weight: 300px;
+}
+
+.firstSearchString:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.imageOne {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.secondSearchString {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  position: relative;
+  width: 100%;
+}
+
+.secondInputLine {
+  font-family: Avenir-Heavy, Arial, Helvetica, sans-serif;
+  font-weight: 400px;
+  border: none;
+  outline: none;
+  padding: 12px 0 10px 33px;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.secondInputLine::placeholder {
```

Comment:
u are duplicating a lot of code, create common classes for elements, and move there all repetitive styles

Reply:
done

Reply:
isn't fixed